example.env

```diff
@@ -1,2 +1,2 @@
-REACT_APP_APIKEY=
+
```

Comment:
Why has this line been removed?

Reply:
I think it because I deleted the file initially, but when I was completing merge conflicts, it went back as it was supposed to be. I haven't edited this file after the commit I have submitted 3 days ago.

Reply:
I think it is still in the files I have submitted, as I have it on my git I have committed and also local

Reply:
https://github.com/MLH-Fellowship/prep-project-23.MAR.PREP.1/blob/5d1f21f518cbb0f7675e0389e40376d95be168cf/example.env

In the last commit it seems to be deleted. Please add it back again

Reply:
I resolved that

Reply:
Yes but now I see the package-lock.json file is still there. Please delete that

Reply:
I have deleted the package